webapp/app/js/perf.js

```diff
@@ -650,13 +650,16 @@ perf.controller('PerfCtrl', [ '$state', '$stateParams', '$scope', '$rootScope',
 
     var optionCollectionMap = {};
 
-    $http.get(thServiceDomain + '/api/optioncollectionhash').then(
-      function(response) {
-        response.data.forEach(function(dict) {
-          optionCollectionMap[dict.option_collection_hash] =
-            dict.options.map(function(option) {
-              return option.name; }).join(" ");
-        });
+    ThOptionCollectionModel.get_list()
+      .success(function(optCollectionData) {
+                 // gather the string representations of option collections
+                 
+                 _.each(optCollectionData, function(optColl) {
```

Comment:
The indentation on these lines isn't correct -- it's too deep and inconsistent with the rest of the file. Please follow the style of the original function.